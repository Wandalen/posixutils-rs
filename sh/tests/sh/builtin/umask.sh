umask
echo $?
umask -S
echo $?

umask 0777
echo $?
umask
umask -S

umask 022
echo test > $TEST_WRITE_DIR/umask.txt
for x in $(ls -l $TEST_WRITE_DIR/umask.txt); do
  echo $x
  break
done
rm $TEST_WRITE_DIR/umask.txt
