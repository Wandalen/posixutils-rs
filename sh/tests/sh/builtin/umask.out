0022
0
u=rwx,g=rx,o=rx
0
0
0777
u=,g=,o=
-rwxr-xr-x
